### ISSUE TITLE
`apt install bat` on Ubuntu 20.04 doesn't add executable to PATH; nor does it install anything

### ISSUE THREAD

**Issue by SamuelMarks** [sharkdp/bat#1186]
After running `apt get install bat` nothing is in PATH; additionally, the suggestion isn't there, i.e.:
```sh
$ bat /etc/apt/sources.list

Command 'bat' not found, but can be installed with:

sudo apt install bacula-console-qt
```

So double-checked, and got nothing:
```sh
$ find / -type f -executable -name bat
```

**Comment by mouse07410** [comment]
Should it not be installed via `cargo install bat`?

I don't think `apt` can do it.

**Comment by sharkdp** [comment]
Please check out the README: https://github.com/sharkdp/bat#on-ubuntu-using-apt

**Comment by SamuelMarks** [comment]
Ahh, `batcat`… thanks

**Comment by richieeinstein1** [comment]
the executable is  `batcat` not `bat`
to use `bat` symlink it this code `mkdir -p ~/.local/bin
ln -s /usr/bin/batcat ~/.local/bin/bat`

**Comment by richieeinstein1** [comment]
the executable is `batcat` not `bat`
to use `bat`. symlink it this code
 `mkdir -p ~/.local/bin`
` ln -s /usr/bin/batcat ~/.local/bin/bat`
And then add `~/.local/bin/bat` to your $PATH

**Comment by Bala2897** [comment]
Not working

**Comment by Bala2897** [comment]
](
![Screenshot_2024-04-04-14-11-28-451_com termux](https://github.com/sharkdp/bat/assets/165627745/e2494207-1a53-4637-97d1-5a98a66906bd)
How to solve this

**Comment by Bala2897** [comment]
unable to execute installed intltool package post-installation script (/data/data/com.termux/files/usr/var/lib/dpkg/info/intltool.postinst): Permission denied               dpkg: error processing package intltool (--configure):                                           installed intltool package post-installation script subprocess returned error exit status 2    Processing triggers for man (1.14.6-1) ...      Errors were encountered while processing:        intltool                                       E: Sub-process /data/data/com.termux/files/usr/bin/dpkg returned an error code (1)

**Comment by Bala2897** [comment]
Anyone pls tell me how to solve this

**Comment by yiqiangjizhang** [comment]
> the executable is `batcat` not `bat` to use `bat`. symlink it this code `mkdir -p ~/.local/bin` ` ln -s /usr/bin/batcat ~/.local/bin/bat` And then add `~/.local/bin/bat` to your $PATH

This solved it in my WSL Ubuntu 22.04

**Comment by eznix86** [comment]
I did it with  `ln -s /usr/bin/batcat /usr/local/bin/bat` for global use
